docs/guide/scala_guide.md

```diff
@@ -135,9 +129,9 @@ println(fory.deserialize(fory.serialize(map)))
 ## Serialize Tuple
 
 ```scala
-val tuple = Tuple2(100, 10000L)
+val tuple = (100, 10000L) //Tuple2
```

Comment:
It is very rare to create tuples by calling the constructor - the Scala compiler has built-in support for tuples and will insert the call for the right tuple class for you